library/std/src/io/mod.rs

```diff
@@ -2066,6 +2057,21 @@ pub trait Seek {
     }
 }
 
+#[doc(hidden)]
+#[unstable(feature = "internal_stream_len_default", issue = "none")]
+pub fn stream_len_default<T: Seek + ?Sized>(self_: &mut T) -> Result<u64> {
```

Comment:
Can't this just be `pub(crate)` rather than `doc(hidden)`?

Reply:
Changed.